engine/src/main/java/org/terasology/rendering/nui/widgets/UIText.java

```diff
@@ -68,27 +68,27 @@
     private TextureRegion cursorTexture;
 
     @LayoutConfig
-    private Binding<String> text = new DefaultBinding<>("");
+    protected Binding<String> text = new DefaultBinding<>("");
 
     @LayoutConfig
-    private boolean multiline;
+    protected boolean multiline;
 
     @LayoutConfig
     private boolean readOnly;
```

Comment:
No, not at all - just change the access modifiers to `protected`! (The reason for this is that a programmer in the future might want to create their own `UIText` override, and to them it might not be obvious why some fields are marked as `protected` but others aren't.)